doc/nrf/protocols/esb/index.rst

```diff
@@ -129,6 +130,17 @@ When the :c:member:`selective_auto_ack` field in the :c:struct:`esb_config` conf
 When the PRX receives a packet that does not require an ACK, it does not send an ACK packet to the PTX.
 In this case, when :c:member:`esb_payload.noack` = ``true``, packet retransmission does not occur.
 
+.. _esb_monitor_mode:
+
+ESB Monitor mode
+================
+
+ESB Monitor mode allows you to listen in on ESB communication between other devices, without sending any packets.
+When ESB is configured as PRX or PTX, it switches between RX, TX, and DISABLE states.
+This causes delays related to radio ramp-up time that is 40 µs when the value of the ``use_fast_ramp_up`` parameter is ``true`` and 130 µs without it.
```

Comment:
what is this, is it a c struct? function? kconfig? I don't see any other usage of it in this PR